cmd/clusterctl/examples/aws/machines.yaml.template

```diff
@@ -21,7 +21,7 @@ items:
   - apiVersion: "cluster.k8s.io/v1alpha1"
     kind: Machine
     metadata:
-      generateName: aws-node-
+      name: aws-node-${CLUSTER_NAME}-1
```

Comment:
I think we should still use `generateName` for nodes, that way creating a new node is simply a copy/paste.